Sources/Amplitude/DefaultTrackingOptions.swift

```diff
@@ -8,20 +9,55 @@ public class DefaultTrackingOptions {
         DefaultTrackingOptions(sessions: false, appLifecycles: false, screenViews: false)
     }
 
-    public var sessions: Bool = true
-    public var appLifecycles: Bool
-    public var screenViews: Bool
-    public var userInteractions: Bool
+    public var sessions: Bool {
+        get {
+            autocaptureOptions.contains(.sessions)
+        }
+        set {
+            if newValue {
+                autocaptureOptions.insert(.sessions)
+            } else {
+                autocaptureOptions.remove(.sessions)
```

Comment:
Why do we remove the session from autocaptureOptions if it's the same value

Reply:
The option might still be in the autocaptureOptions set if sessions is first set to true and then is set to false.